src/functionaltests/java/com/ericsson/ei/query/QueryAggregatedObjectsTestSteps.java

```diff
@@ -296,9 +296,9 @@ public void perform_a_query_for_missed_notification() throws Throwable {
     public void check_missed_notification_has_been_returned() throws Throwable {
         final String expectedResponse = "{\"queryResponseEntity\":{}}";
         final String subscriptionName = "Subscription_1";
-        final String entryPoint = "/queryMissedNotifications";
+        final String entryPoint = "/failed-notifications";
```

Comment:
entryPoint should be a "final static private" global variable as it is used in 2 places.

Reply:
done